### PR TITLE
Processed feedback MS Teams

### PR DESCRIPTION
**Pull request by jeffrey-opdam**
Thank you for your pull request!
By completing this pull request, you agree to the [Contributing License Agreement](https://github.com/ALM-Rangers/VSTS-Bot/blob/master/.github/CLA.md).

Fixes # .

Changes proposed in this pull request:  
- Processed feedback from MS Teams
- 
- 

@ALM-Rangers/VSTS-Bot

### CODE REVIEW

**Comment by jeffrey-opdam** [comment]
Added @dkmsft, I am assuming @Hussar77 will be travelling to the summit as wel..

File: src/VSTS-Bot.Api/Dialogs/ConnectDialog.cs [hunk 1]
```diff
@@ -215,6 +215,14 @@ public ConnectDialog(string appId, string appScope, Uri authorizeUrl, IAuthentic
                 .OrderBy(a => a)
                 .ToList();
 
+            if (!accounts.Any())
+            {
+                reply.Text = Exceptions.NoAccounts;
```
Comment (Hussar77) [inline src/VSTS-Bot.Api/Dialogs/ConnectDialog.cs]: Why do we store normal text response in the Exceptions resource? Looks a little bit inconsistent.

File: src/VSTS-Bot.Api/Dialogs/ConnectDialog.cs [hunk 2]
```diff
@@ -300,6 +308,14 @@ public ConnectDialog(string appId, string appScope, Uri authorizeUrl, IAuthentic
                 .Select(project => project.Name)
                 .ToList();
 
+            if (!this.TeamProjects.Any())
+            {
+                reply.Text = Exceptions.NoTeamProjects;
```
Comment (Hussar77) [inline src/VSTS-Bot.Api/Dialogs/ConnectDialog.cs]: Same comment - why do we store normal text response in the Exceptions resource? Looks a little bit inconsistent.

**Comment by jeffrey-opdam** [comment]
True. I will move it over to the labels

On Mar 5, 2018 23:13, Vladimir Gusarov <notifications@github.com> wrote:

@Hussar77 commented on this pull request.

________________________________

In src/VSTS-Bot.Api/Dialogs/ConnectDialog.cs<https://github.com/ALM-Rangers/DevOps-VSTS-Bot/pull/141#discussion_r172346967>:

> @@ -215,6 +215,14 @@ public ConnectDialog(string appId, string appScope, Uri authorizeUrl, IAuthentic
                 .OrderBy(a => a)
                 .ToList();

+            if (!accounts.Any())
+            {
+                reply.Text = Exceptions.NoAccounts;


Why do we store normal text response in the Exceptions resource? Looks a little bit inconsistent.

**Pull request merged**